<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-A3D81R1" ignored="2" name="Regression suite 1(failed)" tests="4" failures="2" timestamp="16 Jun 2023 14:58:55 GMT" time="0.072" errors="0">
  <testcase name="funC" time="0.012" classname="Testng.TestGDemo3">
    <skipped/>
  </testcase> <!-- funC -->
  <testcase name="funB" time="0.002" classname="Testng.TestGDemo3">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at Testng.TestGDemo3.funB(TestGDemo3.java:13)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- funB -->
  <testcase name="loginTest" time="0.002" classname="Testng.TestNGDemo4"/>
  <testcase name="customerDetails" time="0.009" classname="Testng.TestNGDemo4">
    <failure type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
	at Testng.TestNGDemo4.customerDetails(TestNGDemo4.java:15)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- customerDetails -->
  <testcase name="funD" time="0.0" classname="Testng.TestGDemo3">
    <ignored/>
  </testcase> <!-- funD -->
  <testcase name="funA" time="0.0" classname="Testng.TestGDemo3">
    <ignored/>
  </testcase> <!-- funA -->
</testsuite> <!-- Regression suite 1(failed) -->
